.drone.jsonnet

```diff
@@ -348,6 +348,11 @@ local integration_uefi = Step("e2e-uefi", target="e2e-qemu", privileged=true, de
         "WITH_UEFI": "true",
         "REGISTRY": local_registry,
 });
+local integration_disk_image = Step("e2e-disk-image", target="e2e-qemu", privileged=true, depends_on=[integration_qemu], environment={
```

Comment:
can we make this `depends_on` on `integration_uefi`, so that we serialize those steps, as usually by the time this runs provision tests are still running, and it might be too many tests running in parallel

Reply:
sure